buildscripts/templates/registry/incubating_java/weaver.yaml

```diff
@@ -1,5 +1,5 @@
 params:
-  excluded_namespaces: ["ios", "aspnetcore", "signalr"]
+  excluded_namespaces: ["dotnet", "ios", "aspnetcore", "signalr"]
```

Comment:
Heads up - excluding `dotnet.` attributes as a continuation of https://github.com/open-telemetry/semantic-conventions-java/issues/48